tests/ui/unnecessary_fallible_conversions.rs

```diff
@@ -3,4 +3,16 @@
 fn main() {
     let _: i64 = 0i32.try_into().unwrap();
     let _: i64 = 0i32.try_into().expect("can't happen");
+
+    let _ = i64::try_from(0i32).unwrap();
```

Comment:
Hi, thanks for contributing!

This is a slight nit, but could you add annotations to indicate whether that line should be linted? This helps avoid regressions when running `uibless` to account for new tests.

```suggestion
    let _ = i64::try_from(0i32).unwrap();
    //~^ ERROR use of a fallible conversion when an infallible one could be used
```

There are some examples here: 
https://rustc-dev-guide.rust-lang.org/tests/ui.html#error-annotation-examples

And some details here:
https://rustc-dev-guide.rust-lang.org/tests/ui.html#error-annotations

Reply:
Thank you for reminder! I have added the necessary error annotations and included some additional comments in the tests to make it clearer.